File: src/Prak Pemrograman 1/Pertemuan 5/Tugas Pertemuan 5/PredikatKelulusan.java
import java.util.Scanner;

public class PredikatKelulusan {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Masukan IPK Anda = ");
        double ipk = scanner.nextDouble();
        System.out.println("Masukan lama studi = ");
        double lamastudi = scanner.nextDouble();
        if (ipk >= 3.51 && ipk <= 4 && lamastudi <=4) {
            System.out.println("Selamat Anda Meraih Predikat Summa Cumlaude");
        }else if (ipk >= 3.51 && ipk <= 4 && lamastudi >4)  {
            System.out.println("Selamat Anda Meraih Predikat Cumlaude");
        }else if (ipk >= 3.01 && ipk < 3.51) {
            System.out.println("IPK Anda Sangat Memuaskan");
        }else if (ipk >= 2.76 && ipk < 3.01) {
            System.out.println("IPK Anda Memuaskan");
        }else if (ipk >= 2.00 && ipk < 2.76) {
            System.out.println("IPK Anda Cukup");   
        }else if (ipk < 2.00) {
            System.out.println("Anda Tidak Lulus");
        }else {
            System.out.println("Tidak Valid");
        }
    }
}

File: src/Prak Pemrograman 1/Pertemuan 5/Tugas Pertemuan 5/predikatkelulusan.py
ipk = input("Masukan IPK Anda = ")
ipk = float(ipk)
lamastudi = input("Masukan lama studi = ")
lamastudi = float(lamastudi)
if (ipk >= 3.51 and ipk <=4 and lamastudi <=4):
    print("Selamat Anda Meraih Predikat Summa Cumlaude")
elif (ipk >= 3.51 and ipk <=4 and lamastudi > 4):
    print("Selamat Anda Meraih Predikat Cumlaude")
elif (ipk >= 3.01 and ipk < 3.51):
    print("IPK Anda Sangat Memuaskan")
elif (ipk >= 2.76 and ipk < 3.01):
    print("IPK Anda Memuaskan")
elif (ipk >= 2.00 and ipk < 2.76):
    print("IPK Anda Cukup")
elif (ipk < 2.00):
    print("Anda Tidak Lulus")
else:
    print("Tidak Valid")

